Makefile

```diff
@@ -75,6 +81,12 @@ container-scan: container
 container-public: container
 	docker push ${DOCKER_USER}/observability-stack:${DOCKER_TAG}
 
+checkout-cluster-monitor:
+	@rm -rf microlith/cluster-monitor || true # don't fail the build if it doesn't exist
```

Comment:
I thought -f meant it always succeeded anyway?
```
% ls -l test/abc
ls: test/abc: No such file or directory
% rm -rf test/abc
% echo $?
0
```

Reply:
Separately though the correct way to do this in a makefile is to use a `-` prefix to ignore a failure.